docs/papers/joss/paper.md

```diff
@@ -71,44 +71,72 @@ Some crucial considerations relevant to these packages include:
-library of example user functions that are provided with libEnsemble, maximizing code reuse. For
-instance, users can readily choose an existing generator function and tailor a simulator function
-to their particular needs.
-
 libEnsemble is a complete toolkit that includes generator in-the-loop and backend mechanisms.
 Some other packages cover a sub-set of the workflow. Colmena, for example, has a front-end
 that uses components to create and coordinate tasks while using Parsl to dispatch simulations.
+
+<!--- JLN: I think this should be mentioned in any section that discusses flexibility
 libEnsemble communicates between a manager and multiple workers using either Python's built-in
 multiprocessing, MPI (via mpi4py), or TCP.
+-->
+
+libEnsemble stands out primarily through its generator-simulator paradigm, which eliminates the need
+to explicitly define task dependencies. Instead, it emphasizes data dependencies between
+customizable Python user functions. This modular design lends itself to exploiting the large
+library of example user functions that are provided with libEnsemble or available from its
+community of users. For instance, developers can choose an existing generator function and tailor
+a simulator function to their particular needs.
+
+The close coupling between the libEnsemble generator and simulators enables the generator to perform tasks
+such as asynchronously receiving results, updating models, and canceling previously initiated simulations.
+Simulations that are already running can be terminated and resources recovered. This is more flexible
+compared to other packages, where the generation of simulations is external to the dispatch of a batch 
+of simulations.
 
-libEnsemble takes the philosophy of minimizing required dependencies while supporting various
-back-end mechanisms when needed. For example, the vast majority of users do not require to be
-running a database application or special run-time to use libEnsemble, but for those that do,
-Balsam can be used on the back-end by substituting the regular MPI executor for the Balsam
-executor. This approach simplifies the user experience and reduces the initial setup and
+libEnsemble attempts to minimize required dependencies while still supporting optional
+communication mechanisms and application schedulers. For example, for the vast majority of users
+Python's multiprocessing module is sufficient for parallelism, but for those seeking  
+higher or more complex scaling, simply installing MPI and mpi4py
+allows libEnsemble to run with a MPI runtime without adjusting any code. Most users are likewise satisfied
+scheduling application instances to the same allocation as a libEnsemble instance; but an additional
+pip-install of Balsam and one-line adjust are sufficient for libEnsemble to schedule applications across
+the internet.
+
+<!--- JLN: The below paragraph probably needs some rearranging. I think users don't typically seek
```

Comment:
I dont think I suggested the user wants to run a database, but many workflow tools require it.

Reply:
I prefer my original wording for the most part

Reply:
Ahh, I see, I misunderstood. Feel free to change it back!